### PR TITLE
Dashbord

### PR DESCRIPTION
**Pull request by hamdibm**
i created the dashboard web page

### CODE REVIEW

File: src/app/components/SearchIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
```
Comment (onlymachiavelli) [inline src/app/components/SearchIcon.tsx:1]: Create one file called icons.tsx, put any svg code there, and export all of them as objects : {}

File: src/app/components/topProd.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { Table, TableHeader, TableColumn, TableBody, TableRow, TableCell, Spinner, getKeyValue } from "@nextui-org/react";
+import { useInfiniteScroll } from "@nextui-org/use-infinite-scroll";
+import { useAsyncList } from "@react-stately/data";
+
+export default function TopProd() {
+  type StringNumberKey<T> = Extract<keyof T, string | number>;
```
Comment (onlymachiavelli) [inline src/app/components/topProd.tsx:7]: Stop Copying codes !

File: src/app/page.tsx [hunk 3]
```diff
@@ -1,7 +1,11 @@
 'use client'
+import Donut from './components/DonutChart'
+import MiniDrawer from './components/sideBar'
+import NavBar from './components/NavBar'
+import Dashboard from './dashboard/page';
 const App = () => {
   return (
-      <></>
+      <><Dashboard/></>
```
Comment (onlymachiavelli) [inline src/app/page.tsx:8]: This page is only reserved for the landign page

**Review by onlymachiavelli — CHANGES_REQUESTED** [review]
Please review and fix the requested changes in this commit

File: src/app/auth/login/page.tsx [hunk 4]
```diff
@@ -1,7 +1,71 @@
...
+  disbaled: "#D9D9D9"
 }
+const Login = () => {
+  
+  const router = useRouter();
+
+  const handleSignUp = () => {
+    router.push('/auth/signup');
+  };
+  // const [isClient, setIsClient] = useState(false)
```
Comment (onlymachiavelli) [inline src/app/auth/login/page.tsx]: Remove useless comments

File: src/app/components/topProd.tsx (hunk 2 above)
Comment (hamdibm) [reply src/app/components/topProd.tsx:7]: -it's impossible 
-why???
-just impossible

File: src/app/page.tsx (hunk 3 above)
Comment (hamdibm) [reply src/app/page.tsx:8]: I did that for testing purposes. It's empty now.

**Pull request merged**